src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,167 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> head = null;
+    private Node<T> tail = null;
```

Comment:
it is not necessary to give a null value (null will be specified by default)

Reply:
Good advice!